translation/bn.json

```diff
@@ -0,0 +1,163 @@
+  "Bollinger Bands": "বলিঙ্গার ব্যান্ডস",
+  "Bollinger Bands can be used to measure the highness or lowness of the price relative to previous trades.": "পূর্ববর্তী ট্রেডের সাপেক্ষে মূল্যের উচ্চতা বা নিম্নতা পরিমাপ করতে বলিঙ্গার ব্যান্ডগুলি ব্যবহার করা যেতে পারে।",
+  "Box Size": "বাক্সের আকার",
+  "Cancel": "বাতিল",
+  "Candle": "মোমবাতি",
+  "Channel [num]": "চ্যানেল [num]",
+  "Chart data is not available for this symbol.": "এই প্রতীকের জন্য লেখচিত্রের তথ্য উপলব্ধ নেই।",
+  "Chart types": "লেখচিত্রের ধরন",
+  "Charts": "চার্ট",
+  "Clear all": "সব মুছে ফেলুন",
+  "CLOSED": "বন্ধ",
+  "Color": "রঙ",
+  "Commodity Channel Index": "কমোডিটি চ্যানেল ইনডেক্স",
+  "Continue": "চালিয়ে যান",
+  "Continuous": "একটানা",
+  "Continuous [num]": "ক্রমাগত [num]",
+  "CSV": "সিএসভি",
+  "Dark": "ডার্ক",
+  "day": "দিন",
+  "days": "দিনগুলো",
+  "Detrended Price Oscillator": "ডিট্রেন্ডেড প্রাইস অসিলেটর",
+  "Developed by Larry Williams, Williams %R is a momentum indicator that is the inverse of the Fast Stochastic Oscillator. Also referred to as %R, Williams %R reflects the level of the close relative to the highest high for the look-back period.": "Developed by Larry Williams, Williams %R is a momentum indicator that is the inverse of the Fast Stochastic Oscillator. Also referred to as %R, Williams %R reflects the level of the close relative to the highest high for the look-back period.",
+  "Developed by Tushar Chande in 1995, Aroon is an indicator system that determines whether a stock is trending or not and how strong the trend is. There are two separate indicators: Aroon-Up and Aroon-Down. A 25-day Aroon-Up measures the number of days since a 25-day high. A 25-day Aroon-Down measures the number of days since a 25-day low.": "১৯৯৫ সালে তুশার চান্দে দ্বারা বিকশিত, অরুন একটি নির্দেশক ব্যবস্থা যা নির্ধারণ করে একটি স্টক ট্রেন্ডিং করছে কি না এবং প্রবণতা কতটা শক্তিশালী। দুটি পৃথক সূচক আছে: অ্যারোন-আপ এবং অ্যারোন-ডাউন। একটি 25 দিনের Aroon-up একটি 25 দিনের উচ্চ থেকে দিনের সংখ্যা পরিমাপ। একটি 25 দিনের Aroon-down একটি 25 দিনের কম থেকে দিনের সংখ্যা পরিমাপ।",
```

Comment:
@matin-deriv  please elaborate why do we need to change jsons here? i'm updating translations myself for smartcharts with every release.